radicle-httpd/src/api/tests/scripts/init-e2e-profile

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+
+WORKDIR=/tmp/radicle-httpd-tests/hello-world
+export RAD_HOME=/tmp/radicle-httpd-tests/radicle
+export RAD_PASSPHRASE=asdf
+export RAD_DEBUG=1
+
+mkdir -p $WORKDIR
+mkdir -p $RAD_HOME
+
+export GIT_AUTHOR_NAME="Alice Liddell"
+export GIT_AUTHOR_EMAIL="alice@radicle.xyz"
+export GIT_COMMITTER_NAME="Alice Liddell"
+export GIT_COMMITTER_EMAIL="alice@radicle.xyz"
+export GIT_COMMITTER_DATE=$(date --date="@1673001014" +%Y-%m-%dT%H:%M:%S%z)
+
+cd $WORKDIR
+git init
+echo "Hello World!" > README
+git add .
+git commit -m "Initial commit" --date $GIT_COMMITTER_DATE
+
+mkdir $WORKDIR/dir1
+echo "Hello World from dir1!" > dir1/README
+git add .
+git commit -m "Add another folder" --date $GIT_COMMITTER_DATE
+
+rad auth
+rad init --name "hello-world" --description "Rad repository for tests" --no-confirm
+rad issue new --title "Issue #1" --description "Change 'hello world' to 'hello everyone'"
```

Comment:
there's no way to enforce `GIT_COMMITTER_DATE` to `rad issue new...`, explained further below in tests